src/camd.cpp

```diff
@@ -0,0 +1,99 @@
+#include <chrono>
+#include <ctime>
+#include <frameworkd/classes/daemon/daemon.hpp>
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/videoio.hpp>
+#include <string>
+
+class CamdService : public RoutineService {
+    static cv::Mat frame;
+    static cv::VideoCapture cap;
+    int camTurningOnDelay = 5;
+    std::string absoluteFilePath = "./";
+    int camPhotoDelay = 5000;
+
+    const DBusHandler::Path takePhotoPath {
+        "zfkd.dbus.camd",
+        "zfkd/dbus/camd",
```

Comment:
```suggestion
        "/zfkd/dbus/camd",
```
Isso que da o erro previamente demonstrado anteriormente